code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -137,7 +137,6 @@ Lizard subspecies: ASHWALKERS
 		TRAIT_CAN_STRIP,
 		TRAIT_CHUNKYFINGERS,
 		TRAIT_VIRUSIMMUNE,
-		//TRAIT_LITERATE, ashwalkers are uneducated hillbillies from lavaland 
```

Comment:
This is a code comment to show that the decision to not make them literate was deliberate.

Reply:
Reverted my change then.